### ISSUE TITLE
Broken initial workflow

### ISSUE THREAD

**Issue by atfornes** [bloxberg-org/peer-review-app#19]
When starting the app after commit 4243eb and next one fixing typos the app fail to correctly start.

Redirection or access to '/' path no longer loads the registered user profile.

We tried to debug and fix the issue, but we could not see how :cry:

**Comment by kuzdogan** [comment]
Did you pull the latest comment? There was indeed a typo in 4243eb03c9476da0ac46a4b7a9c5d55c353f9b7a . 

What do you mean by the registered user profile? Do you mean the dashboard/overview? What do you see instead?

**Comment by atfornes** [comment]
Try to enter the app. The user will be presented the registration form all the time, even after sending the registration info. That did not happened before. I think it is because the changes in `npm start client`, but I am not sure.

To reproduce: 
- run the dev server and the client.
- See if you can arrive to the user's profile (it does not work for us even with already registered users)
- It it works, try with a new account and check if it works as well

**Comment by kuzdogan** [comment]
After looking into the breaking commit 4243eb03c9476da0ac46a4b7a9c5d55c353f9b7a I've realized I'm changing the whole contract ABI `ReviewStorage.json` which is caused by me resetting the deployed contract. I do this ofter to create a clean database as we don't have a remove functionality in the contract.  I simply deploy a brand new contract and remove the reviews in the mongodb. 

By executing `truffle migrate --network develop --reset` in your case, we replaced the wrong contract ABI with a new one. 

I guess I should be removing and .gitignoring those files.
